dev/benchmark/add-index-with-load.md

```diff
@@ -0,0 +1,347 @@
+---
+title: Interaction Test on Online Workloads and `ADD INDEX` Operations
+summary: This document tests the interaction effects between online workloads and `ADD INDEX` operations.
+category: benchmark
+---
+
+# Interaction Test on Online Workloads and `ADD INDEX` Operations
+
+## Test purpose
+
+This document tests the interaction effects between online workloads and `ADD INDEX` operations in the OLTP scenario.
+
+## Test version, time, and place
+
+TiDB version: v3.0.1
+Time: July, 2019
+Place: Beijing
```

Comment:
Please add a blank line between these three lines. Otherwise, they'll come into one line at the website.